python/pip_install/tools/wheel_installer/wheel_installer.py

```diff
@@ -32,6 +33,14 @@
 from python.pip_install.tools.wheel_installer import namespace_pkgs, wheel
 
 
+@dataclass
+class Label:
```

Comment:
Can we call this LabelWrapper or something? I just want it to be easier to tell when it's Python code calling the dataclass named Label vs generated bzlcode in the Python code calling the bzl Label. The behavior of Label() is confusing enough; adding "Which label object is this?" is going to make it very confusing.

Actually, can we remove this dataclass entirely? It looks like it's just used once?